src/port/port.tcl

```diff
@@ -4577,6 +4577,15 @@ proc parse_options { action ui_options_name global_options_name } {
                         # debug implies verbose
                         set ui_options(ports_verbose) yes
                     }
+                    d[0-3] {
```

Comment:
This won't work as-is, as argument processing occurs one character at a time. Need to pre-process the arguments string, and ensure the two-character `d<level>` combo is passed through.